plugin-server/src/cdp/services/hog-executor.service.ts

```diff
@@ -401,6 +402,35 @@ export class HogExecutorService {
                         // We need to pass these in but they don't actually do anything as it is a sync exec
                         fetch: async () => Promise.resolve(),
                     },
+                    importBytecode: (module: string) => {
+                        if (module === 'provider/email') {
+                            const provider = this.hogFunctionManager?.getTeamHogEmailProvider(invocation.teamId)
+
+                            if (!provider) {
+                                throw new Error('No email provider configured')
+                            }
+
+                            try {
+                                const providerGlobals = this.buildHogFunctionGlobals({
+                                    id: '',
+                                    teamId: invocation.teamId,
+                                    hogFunction: provider,
+                                    globals: {} as any,
+                                    queue: 'hog',
+                                    timings: [],
+                                    priority: 0,
+                                } satisfies HogFunctionInvocation)
+
+                                return {
+                                    bytecode: provider.bytecode,
+                                    globals: providerGlobals,
+                                }
+                            } catch (e) {
+                                throw new Error(`Error building globals for email provider: ${e}`)
```

Comment:
wrapping the error like this leads to us losing the stack trace.. we should use structured logging here as in 

```suggestion
                                throw new Error(`Error building globals for email provider`, { cause: e })                            }

```